src/MoBi.Presentation/Presenter/EditBuildingBlockPresenterBase.cs

```diff
@@ -85,16 +85,16 @@ private void selectObjectAndContainer(IContainer containerObject, IObjectBase se
 
       public void Handle(EntitySelectedEvent eventToHandle)
       {
-         var selectedObject = eventToHandle.ObjectBase;
-         var (canHandle, containerObject) = CanHandle(selectedObject);
-         if (!canHandle && selectedObject != null)
```

Comment:
This was the issue. This base presenter was handling all `null` selections. That meant that all open presenters would "handle" this by making themselves visible.

We don't want this, instead we want to isolate the null selection to the EditSpatialStructurePresenter where the selection event was raised